guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-smart-proxy-server.adoc

```diff
@@ -14,48 +14,24 @@ For more information, see {InstallingSmartProxyDocURL}registering-to-satellite-s
 For more information, see {InstallingSmartProxyDocURL}installing-capsule-server-packages_{smart-proxy-context}[Installing {SmartProxyServer} Packages].
 
 .Procedure
-. On your {ProjectServer}, validate the custom SSL certificate input files:
+. On your {ProjectServer}, generate a certificate bundle:
 +
+.Example output of `katello-certs-check`
 [options="nowrap", subs="+quotes,attributes"]
 ----
-# katello-certs-check \
--t {certs-proxy-context} \
--c __/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert.pem__ \ <1>
--k __/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert_key.pem__ \ <2>
--b __/root/{smart-proxy-context}_cert/ca_cert_bundle.pem__ <3>
+# {certs-generate} \
+--foreman-proxy-fqdn {smartproxy-example-com} \
+--certs-tar ~/_{smartproxy-example-com}_-certs.tar \
+--server-cert __/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert.pem__ \ <1>
+--server-key __/root/{smart-proxy-context}_cert/{smart-proxy-context}_cert_key.pem__ \ <2>
+--server-ca-cert "__/root/{smart-proxy-context}_cert/ca_cert_bundle.pem__ \ <3>
+--certs-update-server
```

Comment:
@ehelms @evgeni I question if this is still needed (from a technical perspective) but given `katello-certs-check` suggests it and this procedure is for adding certs to an existing proxy I left it in now.